Dockerfiles/agent/entrypoint/50-mesos.sh

```diff
@@ -27,5 +27,5 @@ fi
 if [[ $MARATHON_URL ]] && [[ ! -e $CONFD/marathon.d/conf.yaml.default ]]; then
   mv $CONFD/marathon.d/conf.yaml.example \
      $CONFD/marathon.d/conf.yaml.default
-  sed -i -e "s@# - url: \"https://server:port\"@- url: ${MARATHON_URL}@" $CONFD/marathon.d/conf.yaml.default
+  sed -i -e "s@# - url: \"https://<SERVER>:<PORT>\"@- url: ${MARATHON_URL}@" $CONFD/marathon.d/conf.yaml.default
```

Comment:
Looks like we were expecting the line to be commented `# - url:...` did this change as well?

https://github.com/DataDog/integrations-core/blob/73764c5b60abeea6d88a907f41e0f7b2a2a856b1/marathon/datadog_checks/marathon/data/conf.yaml.example#L13

Reply:
yes, fixing this